lib/srthelper.rb

```diff
@@ -0,0 +1,141 @@
+
+require "json"
+
+SENTENCE_CONTINUATION_DIFF = 0.1
+SINGLE_SRT_SECONDS_LENGTH = 3
+CONTINUATION_SENTENCE_OVERLOAD = 3
+
+
+class SrtHelper
```

Comment:
This class is about parsing Transcribe logic and most appropriate name should be TranscribeHelper rather than SrtHelper.

Reply:
Done